src/core/bpf/meson.build

```diff
@@ -65,13 +65,23 @@ bpf_o_unstripped_cmd += [
         '@OUTPUT@'
 ]
 
-bpf_o_cmd = [
-        llvm_strip,
-        '-g',
-        '@INPUT@',
-        '-o',
-        '@OUTPUT@'
-]
+if bpftool_strip
+        bpf_o_cmd = [
+                bpftool,
+                'g',
+                'o',
```

Comment:
'gen' and 'object' is probably better. It has nothing to do with -g and -o of llvm-strip, btw (pure coincidence).

Reply:
I've changed these to full names in #22330.